tajo-common/src/main/java/org/apache/tajo/type/Type.java

```diff
@@ -36,6 +36,7 @@
 
   // No paramter types
   public static final Any Any = new Any();
+  public static final Bit Bit = new Bit();
```

Comment:
Sorry for the confusion. I'm talking about the modifier of the constructor of the classes which are used in only singleton pattern. I left another comment.